app-modules/prospect/src/Filament/Resources/ProspectResource/RelationManagers/EngagementsRelationManager.php

```diff
@@ -88,18 +88,25 @@ public function infolist(Infolist $infolist): Infolist
                     ->schema([
                         TextEntry::make('deliverable.channel')
                             ->label('Channel'),
-                        IconEntry::make('deliverable.delivery_status')
+                        TextEntry::make('deliverable.delivery_status')
+                            ->iconPosition(IconPosition::After)
                             ->icon(fn (EngagementDeliveryStatus $state): string => match ($state) {
                                 EngagementDeliveryStatus::Successful => 'heroicon-o-check-circle',
                                 EngagementDeliveryStatus::Awaiting, EngagementDeliveryStatus::Dispatched => 'heroicon-o-clock',
                                 EngagementDeliveryStatus::Failed, EngagementDeliveryStatus::DispatchFailed, EngagementDeliveryStatus::RateLimited => 'heroicon-o-x-circle',
                             })
-                            ->color(fn (EngagementDeliveryStatus $state): string => match ($state) {
+                            ->iconColor(fn (EngagementDeliveryStatus $state): string => match ($state) {
                                 EngagementDeliveryStatus::Successful => 'success',
                                 EngagementDeliveryStatus::Awaiting, EngagementDeliveryStatus::Dispatched => 'info',
                                 EngagementDeliveryStatus::Failed, EngagementDeliveryStatus::DispatchFailed, EngagementDeliveryStatus::RateLimited => 'danger',
                             })
-                            ->label('Status'),
+                            ->label('Status')
+                            ->formatStateUsing(fn (Engagement $engagement): string => match ($engagement->deliverable->delivery_status) {
+                                EngagementDeliveryStatus::Successful => 'Successfully delivered',
+                                EngagementDeliveryStatus::Awaiting, EngagementDeliveryStatus::Dispatched => 'Awaiting delivery',
+                                EngagementDeliveryStatus::Failed, EngagementDeliveryStatus::DispatchFailed => 'Failed to send',
+                                EngagementDeliveryStatus::RateLimited => 'Failed to send due to rate limits',
+                            }),
```

Comment:
It's a nit, but if you want I think you could move a lot of this to the enum.

Reply:
I'll add an item to the backlog for this.